File: tree-hugger/lib/tests/fixtures/imports.cs
// Test fixture for C# import extraction
using System;
using System.Collections.Generic;
using System.IO;

// Aliased using
using Console = System.Console;

namespace TestNamespace
{
    public class ImportsTest
    {
        public static void Main()
        {
            List<string> list = new List<string>();
            Console.WriteLine("Hello");
        }
    }
}

File: tree-hugger/lib/tests/fixtures/imports.php
<?php
// Test fixture for PHP import extraction

namespace App\Test;

// Simple use
use App\Models\User;

// Aliased use
use App\Services\AuthService as Auth;

// Function import
use function App\Helpers\format_date;

// Constant import
use const App\Config\VERSION;

class ImportsTest
{
    public function test(): void
    {
        $user = new User();
        $auth = new Auth();
    }
}
